src/Microsoft.ML.TimeSeries/STL/LeastSquares.cs

```diff
@@ -0,0 +1,132 @@
+﻿using System.Collections.Generic;
+using Microsoft.ML.Runtime;
+
+namespace Microsoft.ML.TimeSeries
+{
+    /// <summary>
+    /// this class is used to calculate the least squares of the scatterplots.
+    /// please check http://en.wikipedia.org/wiki/Least_squares for more details.
+    /// </summary>
+    public class LeastSquares
```

Comment:
This class is only used by `LocalRegression`, so please make it `internal` for the reasons explained here:
https://github.com/dotnet/machinelearning/pull/5202/files?file-filters%5B%5D=.cs&file-filters%5B%5D=.csv#r440495855

Reply:
updated, thanks.

---
In reply to: [440505038](https://github.com/dotnet/machinelearning/pull/5202#discussion_r440505038) [](ancestors = 440505038)